src/overlays/actors/ovl_En_Okuta/z_en_okuta.c

```diff
@@ -4,12 +4,24 @@
 
 #define THIS ((EnOkuta*)thisx)
 
+#define DAMAGE_EFFECT_FREEZE 3
+
```

Comment:
the damage effect is something that will be defined by the collision check system at a later time.
you can put a comment next to is usage if you like, but I think its clear based on what function it calls after anyway